cns/hnsclient/hnsclient_windows.go

```diff
@@ -604,6 +609,17 @@ func CreateHostNCApipaEndpoint(
 	return endpoint.Id, nil
 }
 
+// updateGwForLocalIPConfiguration applies change on APIPA NW gw IP address.
```

Comment:
isn't this change resulting into the GW value getting updated for both APIPA NW and APIPA endpoint?